docs/modules/trino/pages/usage_guide/catalogs/generic.adoc

```diff
@@ -0,0 +1,55 @@
+= Generic
+
+As Trino supports a lots of different connectors, we can't possible cover all the available connectors.
+In case the trino-operator does not support a specific connector you can use this connector to pull in any connector Trino supports.
```

Comment:
```suggestion
In case the Trino operator does not support a specific connector you can use this generic connector to utilize any connector Trino supports.
```